### DISCUSSION TITLE
Page builder carousel issue

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#86] [1. Help] [answered]
Hi,

I have created Product carosel from magento backend on homepage. It is working fine. 
Now On particular product image hover, I added product swatches options. 

I have also added carousel for the swatches if there are more than 3 options on product image hover. 

Here when I am clicking on the newt arrow, the whole carousel is being scrolled instead of sliding the option swathes.

Can you please guide me to fix this conflict on double slider of page builder carousel?

You can check here in "best seller" carousel : https://eeupgrade.mothercare.co.id/

![image](https://user-images.githubusercontent.com/9527439/230000921-3a37ada4-485d-420d-818e-07cfaa87bb84.png)

Thanks

**Comment by vovayatsyuk** [comment]
Thanks for reporting. I've created an issue in the breeze module: https://github.com/breezefront/module-breeze/issues/41

**Comment by vovayatsyuk** [comment, marked as answer]
Fixed in repository: https://github.com/breezefront/module-breeze/commit/80544ab66a2f1da0d307a6f04a02b177ff52c399. Will be included in the next release.

**Reply by rutulritz** [reply]
It is working now.

Thanks a lot @vovayatsyuk
